components/post_view/message_attachments/action_button/action_button.test.tsx

```diff
@@ -14,7 +14,7 @@ describe('components/post_view/message_attachments/action_button.jsx', () => {
     const baseProps = {
         action: {id: 'action_id_1', name: 'action_name_1', cookie: 'cookie-contents'},
         handleAction: jest.fn(),
-        theme: Constants.THEMES.default as unknown as Theme,
+        theme: Constants.THEMES.sapphire as unknown as Theme,
```

Comment:
0/5 on doing it now, but maybe this is a good excuse to move the themes into a new file that can use TypeScript so we stop needing to `as unknown as Theme` every time we use those.

Reply:
Actually, maybe we just replace all uses of these with the constants from mattermost-redux?

Reply:
Ya, I was wondering about the seemingly duplicated definitions of themes. Probably should have just asked! I can update to only use the ones from mattermost-redux!